opencti-platform/opencti-front/src/private/components/analysis/groupings/GroupingCreation.js

```diff
@@ -86,6 +86,10 @@ const useStyles = makeStyles((theme) => ({
 const groupingMutation = graphql`
   mutation GroupingCreationMutation($input: GroupingAddInput!) {
     groupingAdd(input: $input) {
+      id
```

Comment:
Why we need to add this lines on fragment creation ?

Reply:
We need this because then node is added in the list, id is required for checkboxes / selection management.